### PR TITLE
[#161] Updated behat/mink-selenium2-driver to 1.7

### LINKED ISSUE #161: Mink-selenium driver release 1.7 introduced BC issues
https://github.com/minkphp/MinkSelenium2Driver/releases/tag/v1.7.0

### PR DESCRIPTION
**Pull request by tannguyen04**
## Checklist before requesting a review

closes #161 

- [x] I have formatted the subject to include ticket number
  as `[#123] Verb in past tense with dot at the end.`
- [x] I have added a link to the issue tracker
- [x] I have provided information in `Changed` section about WHY something was
  done if this was not a normal implementation
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have run new and existing relevant tests locally with my changes, and
  they passed
- [x] I have provided screenshots, where applicable

## Changed

1.

### CODE REVIEW

**Comment by tannguyen04** [comment]
Hi @AlexSkrypnyk 
I have reviewed the change https://github.com/minkphp/MinkSelenium2Driver/compare/v1.6.0...v1.7.0
And looks like only effect to our keyboard.feature which using keyboardTriggerKey function.

File: composer.json [hunk 1]
```diff
@@ -19,8 +19,8 @@
     "require": {
         "php": ">=8.1",
         "behat/behat": "^3",
-        "behat/mink": "<=1.10",
-        "behat/mink-selenium2-driver": "<=1.6",
+        "behat/mink": "<=1.11",
```
Comment (AlexSkrypnyk) [inline composer.json:22]: this constraint should be now saying `>=1.11` and `>=1.7`

**Review by AlexSkrypnyk — CHANGES_REQUESTED** [review]
@tannguyen04 
Please see my comment.

Also, please see changes here https://github.com/minkphp/MinkSelenium2Driver/commit/affb7ea8be3d610e882c565045e797356855ec7b and make sure that correct handling of keys is added to `behat-steps`.

Also, please add some tests to demostrate this.

File: composer.json (hunk 1 above)
Comment (tannguyen04) [reply composer.json]: Hi @AlexSkrypnyk 
I have update constraint as your requested.

I have re-reviewed the changes and handling of keys in our behat-step then could not find anything wrong.
Our behat-step using `withSyn` and `executeJsOnXpath` method on mink selenium, only `withSyn` method change as `Syn` to `syn`.

But I found other thing is the our keyboard trait look like does not handle correct even in main branch. 
Please refer https://github.com/drevops/behat-steps/pull/189

**Pull request merged**